packages/core/__tests__/diff/directive-usage.test.ts

```diff
@@ -724,4 +729,104 @@ describe('directive-usage', () => {
       expect(change.message).toEqual("Directive 'external' was removed from schema 'Foo'");
     });
   });
+
+  describe('directiveUsageFieldAddedFromMeta and directiveUsageFieldRemovedFromMeta', () => {
```

Comment:
I think it might also make sense of having  actualy SDL tests here `diff(a, b)`?